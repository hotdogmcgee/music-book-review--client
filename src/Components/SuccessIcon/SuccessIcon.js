import React from "react";
import './SuccessIcon.scss'


//credit to Petar Stoyanov codepen.io/pepsto
export default function SuccessIcon() {
  return (
    <div className="dummy-positioning d-flex">
      <div className="success-icon">
        <div className="success-icon__tip"></div>
        <div className="success-icon__long"></div>
      </div>
    </div>
  );
}
