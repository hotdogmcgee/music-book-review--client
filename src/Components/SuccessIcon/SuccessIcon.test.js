import React from "react";
import SuccessIcon from "./SuccessIcon";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("SuccessIcon component", () => {
  it("renders a SuccessIcon by default", () => {
    const wrapper = shallow(<SuccessIcon />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
