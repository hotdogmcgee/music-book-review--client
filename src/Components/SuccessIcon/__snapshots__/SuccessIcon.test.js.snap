// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessIcon component renders a SuccessIcon by default 1`] = `
<div
  className="dummy-positioning d-flex"
>
  <div
    className="success-icon"
  >
    <div
      className="success-icon__tip"
    />
    <div
      className="success-icon__long"
    />
  </div>
</div>
`;
